apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabTheorycraft/BuildConstaintCard.tsx

```diff
@@ -0,0 +1,152 @@
+import { StatIcon } from '@genshin-optimizer/gi-svgicons'
+import {
+  CustomNumberInput,
+  CustomNumberInputButtonGroupWrapper,
+  DropdownButton,
+} from '@genshin-optimizer/ui-common'
+import { unit } from '@genshin-optimizer/util'
+import ClearIcon from '@mui/icons-material/Clear'
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  CardHeader,
+  Divider,
+  ListItemIcon,
+  ListItemText,
+  MenuItem,
+  Stack,
+} from '@mui/material'
+import { useContext } from 'react'
+import { ArtifactStatWithUnit } from '../../../../Components/Artifact/ArtifactStatKeyDisplay'
+import CardLight from '../../../../Components/Card/CardLight'
+import type { MinTotalStatKey } from '../../../../Database/DataManagers/CharacterTCData'
+import { minTotalStatKeys } from '../../../../Database/DataManagers/CharacterTCData'
+import { CharTCContext } from './CharTCContext'
+
+export function BuildConstaintCard({ disabled }: { disabled: boolean }) {
+  const {
+    charTC: {
+      optimization: { minTotal },
+    },
+    setCharTC,
+  } = useContext(CharTCContext)
+
+  return (
+    <CardLight>
+      <CardHeader title="Stat Constraints" />
+      <Divider />
+      <Box sx={{ p: 1 }}>
```

Comment:
I think you want CardContent?

Reply:
nope, cardcontent has `2` padding, this needed `1`